uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1022.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1022 {
+  
+  /**
+   * Hide Fields For Contact Summary
+   */
+  public function upgrade_1022() {
+    $valuesRemoved = $this->up1022_getContactEditOptionValues();
+    $optionValues = $this->up1022_getActiveContactEditOptionValues();
+    
+    $activeOptionValues = array_diff($optionValues, $valuesRemoved);
+    civicrm_api3('Setting', 'create', [
+      'contact_edit_options' => $activeOptionValues,
+    ]);
+    
+    return TRUE;
+  }
+  
+  /**
+   * Retrieves active contact edit option setting value
+   *
+   * @return array
+   */
+  private function up1022_getActiveContactEditOptionValues() {
+    $result = civicrm_api3('Setting', 'get', [
+      'sequential' => 1,
+      'return' => ['contact_edit_options'],
+    ]);
+    
+    return $result['values'][0]['contact_edit_options'];
```

Comment:
It's just a personal preference, but I like removing `sequential` and using array_shift on the result instead, what do you think?

Reply:
Worth exploring. My main idea behind sequential is using it whenever i do not care about iterating over entity id. By not using sequential, results will be indexed their `ids` and not array index.